### PR TITLE
Add package for MySQL 8.0 on Ubuntu 20.04

### LINKED ISSUE #442: Add package for MySQL 8.0 on Ubuntu 20.04
We can't use Mroonga on Ubuntu 20.04 without this.

### PR DESCRIPTION
**Pull request by komainu8**
GitHub: fix #442

### CODE REVIEW

**Comment by komainu8** [comment]
The build of Mroonga package for MySQL 8.0 on Ubuntu 20.04 was success.
https://launchpad.net/~groonga/+archive/ubuntu/nightly/+build/22098095

File: packages/mysql-8.0-mroonga/debian/control.in [hunk 1]
```diff
@@ -0,0 +1,103 @@
...
+ It is based on Groonga, a fast fulltext search engine and column store.
+ Groonga is good at real time update.
+ .
+ This is a trnsitional package. This can safely be removed.
+
+Package: mysql-8.0-mroonga
+Section: database
+Architecture: any
+Replaces: mysql-server-8.0-mroonga (<< 11.02-1)
+Breaks: mysql-server-8.0-mroonga (<< 11.02-1)
```
Comment (kou) [inline packages/mysql-8.0-mroonga/debian/control.in:61]:
```suggestion
```

File: packages/mysql-8.0-mroonga/debian/control.in [hunk 2]
```diff
@@ -0,0 +1,103 @@
...
+ It is based on Groonga, a fast fulltext search engine and column store.
+ Groonga is good at real time update.
+ .
+ This is a trnsitional package. This can safely be removed.
+
+Package: mysql-8.0-mroonga-doc
+Section: doc
+Architecture: all
+Replaces: mysql-server-8.0-mroonga-doc (<< 11.02-1)
+Breaks: mysql-server-8.0-mroonga-doc (<< 11.02-1)
```
Comment (kou) [inline packages/mysql-8.0-mroonga/debian/control.in:95]:
```suggestion
```

**Comment by komainu8** [comment]
The build of Mroonga package for MySQL 8.0 on Ubuntu 20.04 was a success.
https://launchpad.net/~groonga/+archive/ubuntu/nightly/+build/22104861

**Pull request merged**